FoodTracker/FoodTracker/MealTableViewController.h

```diff
@@ -9,8 +9,13 @@
 #import <UIKit/UIKit.h>
 #import "Meal.h"
 #import "MealTableViewCell.h"
-#import "ViewController.h"
+#import "MealViewController.h"
 @import os.log;
-@interface MealTableViewController : UITableViewController
-
+@protocol MealTableViewControllerDelegate <NSObject>
+-(void) MealTableViewControllerDidSelectMeal:(Meal *)selectedMeal;
```

Comment:
[apple convention](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/Conventions/Conventions.html) 

> Method names do not have a prefix, and should start with a lowercase letter;

Reply:
or use any other code convention and unify the code style